### PR TITLE
Issue 38: [ENG] Add login/register form inside beat marker

### PR DESCRIPTION
**Pull request by HarisRen**
#### What does this PR do?
Adds functionality of login / register / reset password. Licencer is not used anymore. 

#### What are the relevant issues?
#38

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -150,8 +150,63 @@ <h1>BeatMarker</h1>
...
+				<button class="form-button margin-bot-10" onclick="submitForm('Reset')">Send Reset Instractions</button>
+				<div class="form-text text-right margin-bot-5">Remember Your Password? <a href="#" onclick="changeForm('Login')">Login</a></div>
+				<div class="form-text text-right">Don’t have an account? <a href="#" onclick="changeForm('Register')">Sign up now</a></div>
+			</div>
+		</div>
+		<div id="thankyou-screen" class="register-login-ins modal">
+			<div class="accusonus-logo"><img src="img/accusonusBlack.svg" draggable="false"></div>
+			<div class="thankyou-ins">
+				<div class="thankyou-check"><img src="img/greenCheck.svg" draggable="false"></div>
+				<div id="thankyou-message">Successful registration/login</div>
```
Comment (SpyrosSiannasAccusonus) [inline src/index.html]: Very minor suggestion, we could change this to simply "Success"

File: src/js/index.js [hunk 2]
```diff
@@ -1178,16 +1182,195 @@ function closeLoadingModal(){
...
+function closeThankyouPage(){
+    var modalThankyou = document.getElementById('thankyou-screen');
+    if (modalThankyou.classList.contains('show')){
+        modalThankyou.classList.remove('show');
+    }
+    toggleMainRegLogModal();
+}
+
+// Function that process form requests
+function submitForm(form){
```
Comment (SpyrosSiannasAccusonus) [inline src/js/index.js:1318]: We could also capture the `Enter` key event as well for the form submission.

**Review by SpyrosSiannasAccusonus — APPROVED** [review]
Changes look good. Left two minor comments. You can go ahead if you think we're ok

File: src/index.html (hunk 1 above)
Comment (HarisRen) [reply src/index.html]: Ohh you are right. I had put that as a placeholder and i would like to add the correct text after each action. (After Register -> Successful registration, after Login Successful login). I will correct it. Thanks.

File: src/js/index.js (hunk 2 above)
Comment (HarisRen) [reply src/js/index.js:1318]: Could you please explain a little more, your idea?

Comment (SpyrosSiannasAccusonus) [reply src/js/index.js:1318]: When the user is in the login form, if the user presses `Enter` the form should submit.

Comment (HarisRen) [reply src/js/index.js:1318]: Ohh sorry. Now i understand your point. Thanks.

**Comment by HarisRen** [comment]
@SpyrosSiannasAccusonus I added the changes that you proposed. Please check them out. thanks.

**Review by SpyrosSiannasAccusonus — APPROVED** [review]
Looks awesome 🤘. Merging . . .

**Pull request merged**